# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1730058480 -3600
# Parent  0533954ef115d6d082a7bda8d665b24e43d51c8e
Branding 8: Take copy of installer before signing it.

diff --git a/toolkit/mozapps/installer/windows/nsis/makensis.mk b/toolkit/mozapps/installer/windows/nsis/makensis.mk
--- a/toolkit/mozapps/installer/windows/nsis/makensis.mk
+++ b/toolkit/mozapps/installer/windows/nsis/makensis.mk
@@ -75,8 +75,9 @@ installer:: $(CONFIG_DIR)/setup.exe $(ZI
 	  --setupexe $(CONFIG_DIR)/setup.exe \
 	  --sfx-stub $(SFX_MODULE) \
 	  $(USE_UPX)
 	@echo 'Signing installer.exe.'
+	cp $(ABS_DIST)/$(PKG_INST_PATH)$(PKG_INST_BASENAME).exe $(ABS_DIST)/$(PKG_INST_PATH)$(PKG_INST_BASENAME)-unsigned.exe
 	$(PYTHON3) $(topsrcdir)/browser/installer/windows/sign_installer.py
 ifdef MOZ_STUB_INSTALLER
 	$(PYTHON3) $(MOZILLA_DIR)/mach repackage installer \
 	  -o '$(ABS_DIST)/$(PKG_INST_PATH)$(PKG_STUB_BASENAME).exe' \
